production/helm/loki/templates/monitoring/servicemonitor.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
   labels:
+    instance: primary
```

Comment:
why are we adding this here and removing from pod logs?

Reply:
That's just an error on my side. Thanks 😊